R/misc.R

```diff
@@ -472,3 +472,13 @@ check_case_weights <- function(x, spec) {
   }
   invisible(NULL)
 }
+
+# -----------------------------------------------------------------------------
+check_for_newdata <- function(..., call = rlang::caller_env()) {
+  if (any(names(rlang::enquos(...)) == "newdata")) {
+    rlang::abort(
+      "Did you mean to use `new_data` instead of `newdata`?",
```

Comment:
```suggestion
      "Pleaes use `new_data` instead of `newdata`.",
```
I don't feel particularly strongly about this but it would avoid assuming anything about the user's intent other than which arg they are trying to pass.

Reply:
You got it!